web/src/components/Storyblok/Homepage/HomepageHero/__snapshots__/HomepageHero.test.tsx.snap

```diff
@@ -2,52 +2,56 @@
 
 exports[`Homepage Hero should match snapshot 1`] = `
 <div>
-  <div />
-  <main>
-    <div
-      class="container"
-      data-component="container"
-    >
+  <div
+    class="undefined undefined"
```

Comment:
undefined classes?

Reply:
Embedded fonts from NextJS that haven't been resolved by Jest - nothing to worry about